platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/shadowgraph/SyncLogging.java

```diff
@@ -46,9 +45,7 @@ public static String toShortHashes(List<Hash> hashes) {
         if (hashes == null) {
             return "null";
         }
-        return hashes.stream()
-                .map(h -> CommonUtils.hex(h.getValue(), BRIEF_HASH_LENGTH))
-                .collect(Collectors.joining(","));
+        return hashes.stream().map(h -> h.toHex(BRIEF_HASH_LENGTH)).collect(Collectors.joining(","));
```

Comment:
Not sure how often any of the methods here in `SyncLogging` are called, but if they are called reasonably often might be worth (in the future) rewriting these to _not_ use Java Streams.  

Though really convenient and usually nicely expressive they aren't high performant and generate lots of short-term intermediate objects.  Use for logging you try to minimize that, and if it was noticable a bespoke implementation that did it "brute force" using string builders in a loop would probably be better.

Reply:
this is provided as a lambda to log4j, so they are executed only when a particular marker is turned on, which is only on for debugging